scripts/deploy.js

```diff
@@ -60,6 +43,9 @@ async function deployContract() {
   const tx = await ethStorage.sendValue({ value: hre.ethers.utils.parseEther("0.5") });
   await tx.wait();
   console.log("balance of " + ethStorage.address, await hre.ethers.provider.getBalance(ethStorage.address));
+
+  const startTime = await ethStorage.startTime();
+  console.log("start time is", startTime);
 }
 
 async function updateContract() {
```

Comment:
Can we upgrade the deployed contracts with the new code changed in this PR?

Reply:
No, the slot has changed, so it will be submitted to a new branch.